include/aspect/mesh_refinement/interface.h

```diff
@@ -89,6 +89,19 @@ namespace aspect
         virtual void initialize ();
 
         /**
+         * A function that is called once at the beginning of the refinement
+         * process. The default implementation of the function does nothing,
```

Comment:
That comment is not quite correct (or at least shouldn't be correct). The function should be called once per time step.